### ISSUE TITLE
backend: test_runner - tests fail due to missing "locks" directory

### ISSUE THREAD

**Issue by RoboSparrow** [gardners/surveysystem#182]
Single tests are failing because the `locks` folder is missing in `test_dir`. When running all tests (`sudo ./test_runner ./tests/*`), the *first* test fails with that reason

```
sudo ./test_runner ./tests/answerquestion.test
```

produces

```
* Using /tmp/surveytestrunner.1559021899.16428 as test directory
 * Using 8099 as http port
Stop lighttpd on port 8099...

[FAIL]  answerquestion.test : Answer a question in a survey
Cleaning up...
Stop lighttpd on port 8099...
killing pid 16447 on port 8099...

```

log
```
...
2019/05/28.14:14.36:src/fcgimain.c:392:fcgi_addanswer():Entering page handler.
2019/05/28.14:14.36:src/errorlog.c:54:remember_error():ERROR : Could not open lock file '/tmp/surveytestrunner.1559018676.15086/locks/fb69/lock.fb69bcb4-0000-0000-3b0e-8349056c6410' for append.
2019/05/28.14:14.36:src/errorlog.c:54:remember_error():ERROR : Failed to lock session 'fb69bcb4-0000-0000-3b0e-8349056c6410'
```
[answerquestion.test.log](https://github.com/gardners/surveysystem/files/3225832/answerquestion.test.log)

**Comment by RoboSparrow** [comment]
create missing directory to test bootstrap in test_runner.c::main()

**Comment by RoboSparrow** [comment]
resolved with #183
